File: Phasegate/DayOne/Level 1/python/TEST2/task_two.py
def replace_non_perfect_squares(numbers):    
    result = [0] * len(numbers)
    for count in range(len(numbers)):     
        square_root = 0
        for index in range(1, numbers[count] + 1):
            if index * index <= numbers[count]:
                square_root = index       
        if square_root * square_root == numbers[count]:
            result[count] = numbers[count]
        else:
            result[count] = -1                
    return result

File: Phasegate/DayOne/Level 1/java/TaskTWo.java
public class TaskTwo{
  public static int[] replaceNonPerfectSquares(int[] numbers){
    int[] result = new int[numbers.length];
    for(int count = 0; count < numbers.length; count++){
        int squareRoot = 0;
        for(int index = 1; index * index <= numbers[count]; index++){
            squareRoot = index;
        }
        if(squareRoot * squareRoot == numbers[count]){
            result[count] = numbers[count]; 
        } else{
            result[count] = -1;   
        }
    }
    return result;
  }
}
